kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/shared/service/GwtComponentService.java

```diff
@@ -28,6 +28,21 @@
 public interface GwtComponentService extends RemoteService {
 
     /**
+     * Return the PIDs for all the services that
+     * implements the ConfigurableComponent Maker interface and registered themselves
```

Comment:
*marker

Maybe we can call it generally as findTrackedPids without exposing the underneath implementation.
What do you think?